packages/babel-compat-data/data/plugins.json

```diff
@@ -384,6 +385,6 @@
     "ios": "6",
     "phantom": "2",
     "samsung": "1",
-    "electron": "0.2"
+    "electron": "0.20"
```

Comment:
This is what happens when you consider versions as if they were decimal numbers.